apis/leds/src/tests.rs

```diff
@@ -14,7 +15,7 @@ fn driver_check() {
     let driver = fake::Leds::<10>::new();
     kernel.add_driver(&driver);
 
-    assert!(Leds::count().is_some());
+    assert!(Leds::count().is_ok());
```

Comment:
Should probably do `assert_eq!(Leds::count(), Ok(10));` here. That way, the error code will be printed if the assertion fails.

Reply:
I left it with `is_ok` because it seemed the previous `is_some` was intentional - the contained value is checked below with `assert_eq!(Leds::count().unwrap_or_default(), 10);`

Reply:
Yeah, I'd prefer to switch. It should've been `assert_eq!(..., Some(10));` all along but I missed it in my previous code review.

Reply:
Done - also adjusted below usage.